subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolvePOMIntegrationTest.groovy

```diff
@@ -86,14 +86,15 @@ class ResolvePOMIntegrationTest extends AbstractIntegrationSpec {
         failure.assertHasCause("Could not find lib.pom (project :included-logging:lib)")
     }
 
-    def "resolving a @pom artifact from an included build replacing an external library does not fail the build with a lenient configuration"() {
+    def "resolving a @pom artifact from an included build replacing an external library does not fail the build with a lenient artifact view"() {
```

Comment:
I think we actually want to keep this test as-is until we remove this API because it's explicitly testing that ArtifactView and a lenient configuration do the same thing.

Reply:
Yes, I suppose this is clear if you look at the test class as a whole and aren't just jumping from method to method.  Will restore.

Reply:
This hasnt been restored?